File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        output = []
        maxCandy = max(candies)
        for each in candies:
            if (each + extraCandies) >= maxCandy:
                output.append(True)
            else:
                output.append(False)
        return output

File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        List<Boolean> output = new ArrayList<Boolean>();
        int max = 0;
        for (int each : candies){
            if (max < each)
            max = each;
        }
        for (int each : candies){
            if (each + extraCandies >= max)
            output.add(true);
            else
            output.add(false);
        }
        return output;
        
    }
}